standard/docs/en/schema/changelog.md

```diff
@@ -8,13 +8,15 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ### Advisories
 
 * [#780](https://github.com/open-contracting/standard/pull/780) All extensions have been moved to the [open-contracting-extensions](https://github.com/open-contracting-extensions) organization. No change is required, but we recommend that publishers update the URLs of extensions in release packages and record packages.
-* [#744](https://github.com/open-contracting/standard/pull/744) OCDS now has a [translation and localization policy](http://standard.open-contracting.org/1.1/en/support/governance/#translation-and-localization-policy).
+* [#744](https://github.com/open-contracting/standard/pull/744) OCDS now has a [translation and localization policy](http://standard.open-contracting.org/1.1/en/support/governance/#translation-and-localization-policy). [#808](https://github.com/open-contracting/standard/pull/806) [#808](https://github.com/open-contracting/standard/pull/808) The documentation is now available in Italian.
```

Comment:
PR #806 incorrectly labelled as #808. Also, should the italian translation have it's own bullet point? (currently it follows on from translation and localization policy bullet)

Reply:
Fixed in f3c05cdab953293d305fbb6ca86677d0a3db246c